### ISSUE TITLE
Pivot: Count Distinct

### ISSUE THREAD

**Issue by ctrl-alt-d** [EPPlusSoftware/EPPlus#717]
Hi,

I would like to create a Pivot table with a `Distinct Count`  data field. I guess that this is not supported at this time. Can you confirm?

In Excel, to do a Distinct Count, [it is mandatory to create pivot checking "Add this data to the Data Model" also change "Summarize Values By".](https://answers.microsoft.com/en-us/msoffice/forum/all/cannot-see-distinct-count-in-pivot-table/862a2331-be0a-4a14-8178-d91d0f4675a2)

Congrats about this library, it is awesome.
Best regards.

**Comment by JanKallman** [comment]
No, data models are currently not supported, so the `Distinct Count` will not work with EPPlus. I'll add this as an enhancement for now.
